api/client/lib.py

```diff
@@ -138,6 +138,11 @@ def get_data(url, headers, params=None, logger=None):
         logger = get_default_logger()
         logger.debug(url)
         logger.debug(params)
+    logger.warning('''
```

Comment:
Does anyone have anything against moving this to api/__init__.py

That way we get the warning when we import as opposed to just when you run the file. I added the following to the __init__.py file:

```
from client.lib import get_default_logger
logger = get_default_logger()
logger.warning('''
                Deprecation Warning: Please import from the gro package instead of
                        api.client.api.client will be removed in a future update.
''')
```

Seemed to have the intended effect. Thoughts @nemozen @tolothy @itsjohnward 

![image](https://user-images.githubusercontent.com/1641730/57957119-3c7b8b80-78c9-11e9-8230-e8118783f665.png)

Reply:
also would prevent this warning for triggering every single data call.

Reply:
lgtm. Moved